packages/mui-system/src/cssVars/createGetCssVar.ts

```diff
@@ -10,7 +10,9 @@ export default function createGetCssVar<T extends string = string>(prefix: strin
     const value = vars[0];
     if (
       typeof value === 'string' &&
-      !value.match(/(#|\(|\)|(-?(\d*\.)?\d+)(px|em|%|ex|ch|rem|vw|vh|vmin|vmax|cm|mm|in|pt|pc))/)
+      !value.match(
```

Comment:
The regexp was updated to support `lineHeight` values, for e.g. 1.5 as well as channel values, for e.g. 17 17 17